tools/offcputime.py

```diff
@@ -96,12 +96,11 @@ def signal_ignore(signal, frame):
 BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
 
 int oncpu(struct pt_regs *ctx, struct task_struct *prev) {
-    u32 pid;
+    u32 pid = prev->pid;
```

Comment:
At this point `prev->pid` is "offcpu", `bpf_get_current_pid_tgid` is "oncpu". We update "offcpu"-start-time table:
```c
    if (THREAD_FILTER) {
        ts = bpf_ktime_get_ns();
        start.update(&pid, &ts);
    }
```
so, we really need `prev->pid` here.